src/homepage/howTo/custom-listeners.html

```diff
@@ -0,0 +1,217 @@
+                    </p>
+
+                    <header class="major">
+                        <h3><b>Starting James with BigMessageListener</b></h3>
+                    </header>
+
+                    <p>
+                        When you write a custom Listener, you have to compile it and place the compiled jar package inside "extensions-jars" directory
+                        to make James load your Listener when it starts up. But BigMessageListener is already available in James products.
+                        So we can use it directly without that extra step.
+                    </p>
+
+                    <p>We will take James cassandra product for the example</p>
+
+                    <p>First, get template cassandra product configuration:</p>
+                    <pre><code>
+    $ git clone https://github.com/apache/james-project
+    $ cp -rf james-project/dockerfiles/run/guice/cassandra/destination/conf conf
+    $ mv keystore conf/keystore
+                    </code></pre>
+
+                    <p>
+                        Second, modify listener.xml configuration file in conf/ directory to add BigMessageListener
+                        by specifying its full class name
+                    </p>
+                    <pre><code>
+    &lt;listeners&gt;
+        ...
+        &lt;listener&gt;
+            &lt;class&gt;org.apache.james.examples.custom.listeners.BigMessageListener&lt;/class&gt;
+        &lt;/listener&gt;
+        ...
+    &lt;/listeners&gt;
+                    </code></pre>
+
+                    <p>Finally, starting a James Server by docker compose</p>
+                    <p>Getting James docker-compose.yml</p>
+                    <pre><code>
+    $ wget https://raw.githubusercontent.com/apache/james-project/master/dockerfiles/run/docker-compose.yml
+                    </code></pre>
+                    <p>Add the following volumes for james service:</p>
+                    <pre><code>
+    volumes:
+      - $PWD/conf:/root/conf/
+                    </code></pre>
+
+                    <p>
+                        If you are using your custom listeners, place the jar containing your listeners into "extensions-jars" directory
+                        and add a volume for it
+                    </p>
+                    <pre><code>
+    volumes:
+      - $PWD/extensions-jars:/root/extensions-jars/
+                    </code></pre>
+                    <p>
+                        Just like that, with a very simple preparation, you can make James use your custom listeners to serve your purpose.
+                    </p>
+                </div>
+                <footer class="major">
+                    <ul class="actions align-center">
+                        <li><a href="index.html" class="button">go back to other how-tos</a></li>
+                    </ul>
+                </footer>
+            </div>
+        </section>
+
+    </div>
+    <footer id="footer" class="major">
```

Comment:
this doesn't make sense to me, we use Jekyll to avoid such repetition, I don't get why we write html as if we were not using Jekyll.
Let's figure this out later but we should handle that maybe tomorrow

Reply:
I have no idea about it

Reply:
What @mbaechler means is that with Jekyll server's templates and layouts we shouldn't need to write the header and the footer in every html file of our website. I just pushed a little quick refactor with a layout for howTo pages regarding this